src/formatting/modules/visitor.rs

```diff
@@ -49,9 +49,9 @@ impl<'a, 'ast: 'a> CfgIfVisitor<'a> {
         // extern crate cfg_if;
         // cfg_if! {..}
         // ```
-        match mac.path.segments.first() {
-            Some(first_segment) => {
-                if first_segment.ident.name != Symbol::intern("cfg_if") {
+        match mac.path.segments.last() {
```

Comment:
I assume the first to last change will still allow support for `cfg_if::cfg_if!` and  `cfg_if!` correct?

Reply:
Yes, I added to support self-importing like `pub use cfg_if::cfg_if;` in the root file and using cfg_if! as `crate::cfg_if!`.

Reply:
Even better! Changes LGTM, CI failures seem to just be due to the configuration snippet tests needing lazy_static

Reply:
Thanks for the catch! Fixed in https://github.com/rust-lang/rustfmt/pull/4268/commits/5c7cb944697426a8e6139c438e457c91e14cec79.